spec/kong_tests.conf

```diff
@@ -4,7 +4,6 @@ proxy_listen = 0.0.0.0:9000
 proxy_listen_ssl = 0.0.0.0:9443
 cluster_listen = 0.0.0.0:9946
 cluster_listen_rpc = 127.0.0.1:9373
-ssl = on
```

Comment:
That shouldn't be removed (from the base config and the provided default config). It should be possible to run Kong without SSL support. Default can be `on` if desired.